etc/neutron/plugins/ml2/ml2_conf_cisco_apic.ini

```diff
@@ -0,0 +1,61 @@
+[ml2_cisco_apic]
+
```

Comment:
This sample config file is misleading - don't think any of these options are applicable to the host-agent. We should put a description and default values for options are applicable in the sample file.

Reply:
Yeah -- there are just a handful:

https://github.com/noironetworks/python-opflex-agent/blob/master/opflexagent/apic_topology.py#L54-L64

I'll update this accordingly.

Reply:
You might also consider naming the config file section to something other than "ml2_cisco_apic"

Reply:
How about "apic_host_agent"? None of these are shared with anything.

Reply:
sure